src/scripts.js

```diff
@@ -233,9 +256,105 @@ function bookRoom(event) {
     addBookingsPost()
     updateAvailableContainer()
     updatePastAndUpcomingBookingsContainer()
+    updateGuestAvailableBookings()
+    updateGuestPastAndUpcomingBookings()
+    calculateClientExpenses()
+}
+
+function currentDate() {
+    let date = new Date().toLocaleDateString()
+    const splitDate = date.split("/")
+    const reverseDate = splitDate.reverse()
+    const joinDate = reverseDate.join('/')
+    // console.log(reverseDate)
+    currentDay = joinDate
+}
+
+function findGuest() {
+    currentClient = clients.find(client => client.name.toLowerCase() === findGuestInput.value.toLowerCase())
+    currentClient.determineUserPastBookings()
+    updateGuestPastAndUpcomingBookings()
     calculateClientExpenses()
 }
 
+function updateGuestAvailableBookings() {
+    guestAvailableRoomsContainer.innerHTML = ''
+    currentClient.filteredBookings.forEach(filteredBooking => {
```

Comment:
Question: You're  returning almost the same bit of markup in more than one place. Could you create and invoke functions. to dry up your code?